src/harvest.js

```diff
@@ -368,8 +368,12 @@
     });
   }
 
-  $(function() {
+  function removeSubmitButton() {
```

Comment:
While this may be convention in some companies, I don't quite think this balance extension should go modifying Harvest outside balance functionality? Or at least make it optional (and leave UI as-is by default)